src/main/flight/pid.c

```diff
@@ -93,6 +93,21 @@ void pidResetErrorGyro(void)
 
 const angle_index_t rcAliasToAngleIndexMap[] = { AI_ROLL, AI_PITCH };
 
+
+// Low Pass filter for PID controllers
+void pt1_element(int axis, float *delta, uint8_t f_cut, int type) {
+
+	float dT = (float)cycleTime * 0.000001f;
+
+    float    RC;
+    static float    lastDelta[][3] = {{ 0.0f, 0.0f, 0.0f }};
+
```

Comment:
What you are saying is only happening when not defined as static in the previous situation.